seclang/parser.go

```diff
@@ -85,27 +85,27 @@ func (p *Parser) FromString(data string) error {
 		} else if inQuotes && len(line) > 0 && line[0] == '`' {
```

Comment:
Not related to this PR, but I'm not able to grok the `inQuotes` logic well. It seems to show that a starting quote is always the last character of a line and ending quote is always the first, is that how it works? 

If anyone can point me at an example rule for the inQuotes logic that would be great.